client/components/common/ProgressDialog.vue

```diff
@@ -8,7 +8,7 @@
         <v-card-text>
           Please wait...
           <v-progress-linear
-            indeterminate
+            v-model="status"
```

Comment:
If progress is controlled via prop then it might make sense to bind it to `:value` instead of using v-model